jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -363,6 +393,24 @@ def _display_args_changed(self, msg={}):
                 mark.colors = [self.color]
                 mark.fill_opacities = [self.fill_opacity]
 
+    def import_region(self, region):
+        """
+        Import an astropy regions object (or file).
+
+        Parameters
+        ----------
+        region : str or regions.Regions object
+        """
+        if isinstance(region, (regions.Region, regions.Regions)):
+            self.preset.import_obj(region)
+        elif isinstance(region, str):  # TODO: support path objects?

```

Comment:
Duy would strongly be in favor of Path support, yes, heh heh.

Reply:
would users want to provide a `Path`?  This isn't ever used internally and is strictly user-facing.... but if they would, it should just be a matter of adding to the `isinstance` and casting to a string either her or in `preset.import_file` (it must be a string by the time it hits the select component, I think).  I just didn't want to overcomplicate the logic unless its something we want.  As it is now, a `Path` object will raise a sensible error and the user could cast to string.